pkgs/development/tools/codespelunker/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "codespelunker";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "boyter";
+    repo = "cs";
+    rev = "v${version}";
+    hash = "sha256-NN/78paePdvYHQ4J2aQu56PvEciOXY8DxHd4ajfVCFU=";
+  };
+
+  vendorHash = null;
+
```

this removes the extra `csperf` and `parser` binaries which doesn't seem to be part of the main program

```suggestion

  subPackages = [ "." ];

```